src/payment/strategies/cardinal/cardinal.ts

```diff
@@ -69,7 +71,7 @@ export interface CardinalValidatedData {
     ActionCode?: CardinalValidatedAction;
     ErrorDescription: string;
     ErrorNumber: number;
-    Validated: boolean;
+    Validated?: boolean;
```

Comment:
IDK for v1, it doesn't allow to close the modal in test mode 🤷🏻‍♂️ In v2 the response object is not including this property even when documentation says it will be. That's why I made it optional.
![Screen Shot 2020-11-04 at 3 14 49 PM](https://user-images.githubusercontent.com/4843328/98404099-99c04b00-202f-11eb-81f8-10d6de13fb49.png)